src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -140,12 +141,14 @@ class Chttp2ServerListener : public Server::ListenerInterface {
       grpc_closure on_receive_settings_ ABSL_GUARDED_BY(&connection_->mu_);
       grpc_pollset_set* const interested_parties_;
       grpc_resource_user* channel_resource_user_;
+      std::vector<const grpc_channel_filter*> filters_;
```

Comment:
Channel filter is a type that'd used above the transport layer and should not be added as a thing that transports need to deal with.

If we need to pass filters through this code, suggest creating a new type that is *opaque* to the transport code and can be manipulated only via higher level types.